documentation/release_notes.rst

```diff
@@ -8,6 +8,64 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.3.0
+===============
+
+New Features
+------------
+- Added dynamic selection as an experimental feature. Provides functions such as ``select``,
+  ``submit`` and ``submit_and_wait``. It includes selection policies: ``fixed_resource_policy``,
+  ``round_robin_policy``, ``dynamic_load_policy``, ``auto_tune_policy``, and provides default
+  support for SYCL queues. 
+- Algorithms using vector execution policies are now vectorized whenever possible when the program is built with Intel® DPC++/C++ Compiler.
+- Added support for passing zip iterators as segment value data in ``reduce_by_segment``, ``exclusive_scan_by_segment``,
+  and ``inclusive_scan_by_segment``.
+- Improved performance of ``merge``, ``sort`` and ``stable_sort`` algorithms on GPU devices.
```

Comment:
@MikeDvorskiy, does it also affect `sort_by_key`?

Reply:
It may affect in case of non-arithmetic key.

Reply:
So, the phase may be extended as
 Improved performance of ``merge``, ``sort`` , ``stable_sort`` and ``sort_by_key`` algorithms on GPU devices.